specs/deposits.md

```diff
@@ -193,6 +193,9 @@ A predeployed contract on L2 at address `0x4200000000000000000000000000000000000
 certain block variables from the corresponding L1 block in storage, so that they may be accessed
 during the execution of the subsequent deposited transactions.
 
+A `seqnr` attribute is stored along with the L1 attributes to track the L2 block distance
+to the L2 block height that the L1 attributes were first introduced.
```

Comment:
@tynes spec is already updated. Suggestions welcome if you want to word it differently.

Reply:
I was thinking that your wording above was much more clear :)

It resets to 0 whenever the L1 origin fields change. This allows it to uniquely identify L2 blocks in the same epoch, without needing the context of the actual L2 block height.